tests/candev/main.c

```diff
@@ -58,6 +58,12 @@ static candev_mcp2515_t mcp2515_dev;
 static isrpipe_t rxbuf;
 static uint8_t rx_ringbuf[RX_RINGBUFFER_SIZE];
 
+#define CAN_RX_FILTER_MASK_EXT   0x9FFFFFFF
+#define CAN_RX_FILTER_MASK_STD   0x7FF
+#define CAN_RX_FILTER_ID_T1     0x001
+#define CAN_RX_FILTER_ID_T2     0x8C11FFFF
```

Comment:
What I meant was: How is this value composed?

What are the bits / fields that create this value - so like

```C
filter[0].can_id = CAN_RX_FILTER_FOO | CAN_RX_FILTER_BAR | CAN_RX_FILTER_BAZ;
```